uberenv.py

```diff
@@ -585,6 +585,33 @@ def __init__(self, opts, extra_opts):
 
         print("[spack spec: {0}]".format(self.opts["spec"]))
 
+        # List of concretizer options not in all versions of spack
+        # (to be checked if it exists after cloning spack)
+        self.fresh_exists = False
+        self.reuse_exists = False
+
+    def check_for_fresh(self):
```

Comment:
Suggestion: Please consider renaming this function to:
```suggestion
    def check_concretizer_opts(self):
```

Reply:
Good catch... pushing in a bit